lib/plugins/aws/deploy/compile/events/apiGateway/README.md

```diff
@@ -96,6 +96,25 @@ functions:
 **Note:** The templates are defined as plain text here. However you can also reference an external file with the help of
 the `${file(templatefile)}` syntax.
 
+#### Pass Through Behavior
+API Gateway provides multiple ways to handle requests where the Content-Type header does not match any of the specified mapping templates.  If not specified, a value of *NEVER* will be used.
```

Comment:
Can you also mention the `415 - Unsupported Media Type` message users will get when there is no template defined for their content-type. This way if they search the docs its easy for them to find it this setting and set it.